setup.py

```diff
@@ -9,12 +12,35 @@
             break
 
 DISTNAME = 'matlab_kernel'
+
+# generating kernel.json for both kernels
+os.makedirs(os.path.join(DISTNAME, 'matlab'), exist_ok=True)
```

Comment:
I think we need to add these paths to `.gitignore` so we don't accidentally add them to source control at some point.   We can also rename the existing `kernel.json` to `kernel_template.json` to avoid confusion.

Reply:
That's a good point. It's fixed in a new commit.

Reply:
Can you please update `MANIFEST.in` as well to make sure the template ends up in the sdist?

Reply:
Sure, I'm not familiar with `MANIFEST.in`, hope I did it right. Also I pushed a rebase to squash the 4 small commits into one.